common/models/role-mapping.js

```diff
@@ -5,6 +5,7 @@
 
 'use strict';
 var loopback = require('../../lib/loopback');
+var utils = require('../../lib/utils');
 
```

Comment:
all changes related to promisifying RoleMapping model are covered in https://github.com/strongloop/loopback/pull/3169